readme.md

```diff
@@ -2,3 +2,23 @@ Nextras ORM Plugin for PhpStorm
 ===============================
 
 Provides support for [Nextras ORM](https://github.com/nextras/orm) � an ORM library for PHP.
+
+## Contributing
+
+### Provisioning
+
+- install [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/)
+- install PHP plugin
+- Import Project, Create project from existing sources, keep defaults (seemingly empty project will be created)
+- File, Project Structure, Modules, click +, New Module
```

Comment:
for the reference https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems